lib/compliance/compliance-tiers.js

```diff
@@ -1,18 +1,20 @@
-const _ = require('lodash/fp')
-
 const { REQUIREMENTS } = require('./triggers/consts')
-
+const uuid = require('uuid')
```

Comment:
This should be separated from local file imports

```suggestion
const uuid = require('uuid')

const { REQUIREMENTS } = require('./triggers/consts')
```